pkg/agent/util/net_windows.go

```diff
@@ -897,21 +898,40 @@ func VirtualAdapterName(name string) string {
 	return fmt.Sprintf("%s (%s)", ContainerVNICPrefix, name)
 }
 
+func MoveIFConfigurations(ips []*net.IPNet, routes []interface{}, mac net.HardwareAddr, mtu int, from string, to string) error {
```

Comment:
Why this function is empty?

Reply:
We are relying on New-VmSwitch (Windows) to create virtual adapter for us

Reply:
I think it works on Windows because we have these preconditions/assumptions: 1) there is only one network adapter on the Windows VM, 2) the update on ExternalNode operation doesn't change the expected network adapter although we only process one interface in ExternalNode. If user's configuration breaks any of the above assumption, antrea-agent on Windows would not work, right?

So I think we should implement this finally. If you have other plan to implement it in a future change, please add comment here to explain in what condition the empty function does not impact the result, and add TODO. Otherwise, please provide the implementation along in this change.

Reply:
Discussed offline. Decided to add TODO in this function.